### DISCUSSION TITLE
Any chance of a portable version?

### DISCUSSION THREAD

**Discussion by BlohoJo** [radubirsan/FlashBrowser#22] [Ideas]
As the title says! :D

**Comment by radubirsan** [comment]
good idea

**Comment by PastelCake888** [comment]
Does this work on the phone ?

**Reply by ArtNFT-ESDT** [reply]
No yet!

**Reply by ArtNFT-ESDT** [reply]
Flash Browser is for  Windows SO .
A simple and fast browser, with a Special dedicated Menu for Flash Games.
